src/Layout/layout-styles.js

```diff
@@ -0,0 +1,135 @@
+// @flow weak
+import { reduce, range, flow, partialRight } from 'lodash/fp';
+
+const baselineGrid = 8;
+
+const generateProps = (propSetter, from, to) => flow(
+  range,
+  reduce((a, i) => ({
+    ...a,
+    ...propSetter(i),
+  }), { }),
+)(from, to);
+
+const flexOrder = () => ({
+  flexOrder: { order: 0 },
+  ...generateProps((i) => ({ [`flexOrder${i}`]: { order: i } }), -20, 20),
+});
+
+const offset = () => ({
+  ...generateProps((i) => ({ [`flexOffset${i}`]: { marginLeft: `${i * 5}%` } }), 0, 19),
+  flexOffset33: { marginLeft: 'calc(100% / 3)' },
+  flexOffset66: { marginLeft: 'calc(200% / 3)' },
+});
+
+const boxSizing = 'border-box';
+const generateFlexProps = partialRight(generateProps, [0, 20]);
```

Comment:
Again, I'm wondering about this `20` granularity, having a smaller set of possibilities could enforce having a more consistent UI.